SPHINXsys/src/for_3D_build/high_level_simulation_class/solid_structural_simulation_class.cpp

```diff
@@ -221,7 +224,7 @@ void StructuralSimulation::ScaleTranslationAndResolution()
 void StructuralSimulation::SetSystemResolutionMax()
```

Comment:
for function name, using minor first letter of the verb?

Reply:
done